BUILD.md

```diff
@@ -102,151 +247,231 @@ If you want a list of the available Python tests:
 $ pytest cuML/test --collect-only
 ```
 
-### Manual Process
+### Full Process
 
-Once dependencies are present, follow the steps below:
+### Building C++ Artifacts with CMake
```

Comment:
This header seems to be at the same nested level as the one right before it. Was that intentional?